sql/src/main/java/io/crate/analyze/CreateAnalyzerAnalyzedStatement.java

```diff
@@ -67,6 +71,10 @@ public void ident(String ident) {
             throw new IllegalArgumentException(String.format(Locale.ENGLISH,
                     "Cannot override builtin analyzer '%s'", ident));
         }
+        String underscoreIdent = Strings.toUnderscoreCase(ident);
```

Comment:
because on the deprecation message we give a naming suggestion (`use [my_name] instead`)